### ISSUE TITLE
ImportError: cannot import name '_FEATURES'

### ISSUE THREAD

**Issue by hlra** [cbjrobertson/ftr_classifier#1]
When trying to import library, I get the following error:

`ImportError: cannot import name '_FEATURES'`

Possibly related, there are also issues downloading and linking the spacy english language model during the installation of the package as described here:
https://stackoverflow.com/questions/54334304/spacy-cant-find-model-en-core-web-sm-on-windows-10-and-python-3-5-3-anacon

I trying to run the package in a Conda environment with Python 3.6 as described in the readme.

**Comment by cbjrobertson** [comment]
Can you add a minimal reproducible example to this issue please?

**Comment by hlra** [comment]
Sure. Sorry for being unlcear. The only lines of code I run are the ones from the readme.

In my Python terminal:
`conda create --name my_env_name python=3.6`
`conda activate my_env_name`
`pip install ftr-classifier`

After that, in the new environment:
`import ftr_classifier as ftr`

The import throws the error.

**Comment by cbjrobertson** [comment]
I was unable to reproduce that error, but did successfully install the `ftr-classifier` into a new `conda`  environment using the following steps. I ran into the same error linking `spaCy`, which I believe relates to different model import requirements `spaCy 3.x`. I solved these issues by installing `spaCy 2.2.1`.  I know I should update the `ftr-classifier` for contemporary compatibility, but I'm not sure I'll have the time to work on it in the near future. In the mean time, try the steps below.

```
conda create -n my_env_name python=3.6
conda activate my_env_name
conda install -c conda-forge spacy=2.2.1
pip install ftr-classifier
```

Then, in `python`, run: 

`import ftr_classifier as ftr`
